docker-compose.yml

```diff
@@ -30,3 +30,14 @@ services:
       - django
     ports:
       - "80:80"
+
+  render:
+    build:
```

Comment:
We seems the docks are woefully incorrect, when I have what you suggest it tells me file not found. I'll fiddle with this more tomorrow.